src/java.desktop/unix/native/common/awt/awt_GraphicsEnv.h

```diff
@@ -62,4 +62,11 @@ struct X11GraphicsConfigIDs {
     jfieldID bitsPerPixel;
 };
 
+#define MAX_DISPLAY_MODES 256
```

Comment:
Copyright year needs to be updated for this file.

Reply:
It can be handled as a batch copyright year update.